tests/helpers/misc.test.ts

```diff
@@ -198,3 +198,29 @@ describe("lazy", () => {
     expect(lazy(5).map((n) => n + 1)()).toBe(6);
   });
 });
+
+describe("isConsecutive", () => {
+  test("consecutive set", () => {
+    const consecutiveSet = new Set<number>();
+    consecutiveSet.add(2);
+    consecutiveSet.add(3);
+    consecutiveSet.add(1);
+    expect(isConsecutive(consecutiveSet)).toBeTruthy();
```

Comment:
Wouldn't it be more readable with an array ?
```suggestion
  test("consecutive", () => {
    expect(isConsecutive([2, 3, 1])).toBeTruthy();
```